packages/scripts/README.md

```diff
@@ -0,0 +1,15 @@
+# @wordpress/scripts
+
+A collection of JS scripts for WordPress development.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/scripts --save
```

Comment:
The PR summary contains the line: "To include you would have to include in devDependencies only this:"

 Would `--saveDev` make more sense?